functions/pipeline/images/__init__.py

```diff
@@ -0,0 +1,63 @@
+import logging
+
+import azure.functions as func
+import json
+
+from ..shared.vott_parser import create_starting_vott_json
+from ..shared.db_provider import get_postgres_provider
+from ..shared.db_access import ImageTagDataAccess, ImageTagState
+
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    logging.info('Python HTTP trigger function processed a request.')
+
+    image_count = int(req.params.get('imageCount'))
+    user_name = req.params.get('userName')
+    tag_status = req.params.get('tagStatus')
+
+    # setup response object
+    headers = {
+        "content-type": "application/json"
+    }
+    if not user_name:
```

Comment:
@samiyaakhtar one scenario might be that the CLI calls our Labels API and gets a list of tags that include the image Id. Next the CLI calls this Images API and does a GET call with the list of image Ids as a param. This Images API returns a list of Image Info metadata (url, height width, etc). What do you think about supporting this scenario?